benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py

```diff
@@ -470,16 +478,18 @@ def initialize_workload(self):
         if self._odf_pvc == 'True':
             self.odf_pvc_verification()
         self._template.generate_yamls()
-        if self._enable_prometheus_snapshot:
+        if self._enable_prometheus_snapshot == 'True':
             self.start_prometheus()
 
     def finalize_workload(self):
         """
         This method includes all the finalization of workload
         :return:
         """
-        if self._enable_prometheus_snapshot:
+        if self._enable_prometheus_snapshot == 'True':
             self.end_prometheus()
         if self._endpoint_url:
             self.upload_run_artifacts_to_s3()
+        if self._save_artifacts_local == 'False':
```

Comment:
Compare everything against "True" or "False", not some things against one and some things against the other (what happens if the value is something different?).

Reply:
Not needed.